tests/util/utils.ts

```diff
@@ -24,6 +25,11 @@ export function toBN(value: BigNumberish) {
   return number.toBN(value);
 }
 
+// Convert a BigNumberish to a 32 byte uint array
+export function numberToU832Bytes(value: number) {
```

Comment:
This [comment](https://github.com/keep-starknet-strange/madara/pull/474/files#r1207781425) explains why this utility was added.